Gosu/Window.hpp

```diff
@@ -111,8 +111,11 @@ namespace Gosu
         //! in your implementation.
         virtual void button_down(Gosu::Button);
         
-        //! Same as button_down. Called then the user releases a button.
+        //! Same as button_down. Called when the user releases a button.
         virtual void button_up(Gosu::Button) {}
+
+        //! Called when you (drag &) drop a file on the window. Serves the path of the dropped file as argument.
+        virtual void drop(char* path) {}
```

Comment:
Hehe. I actually tried that first but I got conversion errors :cry: But I'll try again needed quite a few runs from scratch to make it work so the error might have been cause by something else.